scenarios/platform.benchmarks.yml

```diff
@@ -17,8 +17,10 @@ jobs:
     readyStateText: Application started.
     framework: net5.0
     environmentVariables:
+      DOTNET_SYSTEM_NET_SOCKETS_INLINE_COMPLETIONS: 1
+      DOTNET_SYSTEM_NET_SOCKETS_THREAD_COUNT: 1
       database: PostgreSQL
-      connectionstring: Server={{databaseServer}};Database=hello_world;User Id=benchmarkdbuser;Password=benchmarkdbpass;Maximum Pool Size=28;Enlist=false;Max Auto Prepare=4;Multiplexing=true;Write Coalescing Buffer Threshold Bytes=1000
+      connectionstring: Server={{databaseServer}};Database=hello_world;User Id=benchmarkdbuser;Password=benchmarkdbpass;Maximum Pool Size=20;Enlist=false;Max Auto Prepare=4;Multiplexing=true;Write Coalescing Buffer Threshold Bytes=1400
```

Comment:
Would love to see some comments with how these numbers were chosen so we have an idea on how to check if they are still the most appropriate.